scripts/Muon/GUI/Common/fitting_tab_widget/fitting_tab_model.py

```diff
@@ -174,12 +175,28 @@ def do_single_tf_fit(self, parameter_dict):
 
     def do_single_fit_and_return_workspace_parameters_and_fit_function(
             self, parameters_dict):
-        alg = mantid.AlgorithmManager.create("Fit")
+        if 'DoublePulseEnabled' in self.context.gui_context and self.context.gui_context['DoublePulseEnabled']:
```

Comment:
why do we enable double pulse in 2 different places?

Reply:
There are two paths through the code taken when a fit is done depending on whether it is a single or simultaneous fit. We need to enable double pulse in both these pathways.

Reply:
ah ok, that was not obvious to me